inbm/fpm/mqtt/template/etc/intel-manageability/public/mqtt-broker/acl.file

```diff
@@ -17,11 +17,11 @@ topic write dispatcher/query
 user cloudadapter-agent
 topic cloudadapter/#
 topic write manageability/request/+
-topic write TopicTelemetryInfo
+topic write uccctl/tel/req/#
```

Comment:
Isnt the uccctl/# topic for the UCC MQTT broker?
Do we need to list the external CSP's topic's here?

Reply:
> Isnt the uccctl/# topic for the UCC MQTT broker? Do we need to list the external CSP's topic's here?

external CSP topics should not be listed in this file